planning_scene/src/planning_scene.cpp

```diff
@@ -1133,8 +1133,10 @@ void planning_scene::PlanningScene::decoupleParent()
   parent_.reset();
 }
 
-void planning_scene::PlanningScene::setPlanningSceneDiffMsg(const moveit_msgs::PlanningScene &scene_msg)
+bool planning_scene::PlanningScene::setPlanningSceneDiffMsg(const moveit_msgs::PlanningScene &scene_msg)
 {
+  bool ret = true;
```

Comment:
Minor but, MoveIt!'s code style is to write out variable names in full form as much as possible. ``bool result`` is very common throughout the code base, so I think this variable should also be called ``result``, thanks!